commons/src/main/java/io/hyscale/commons/models/DockerCredHelper.java

```diff
@@ -60,11 +60,14 @@ public CredsStoreEntity get(String registryUrl) {
         try {
             CommandResult result = CommandExecutor.executeAndGetResults(command, registryUrl);
             ObjectMapper mapper = ObjectMapperFactory.jsonMapper();
-            if(result!=null && StringUtils.isNotBlank(result.getCommandOutput())&& result.getExitCode()!=1) {
-                CredsStoreEntity credsStore = mapper.readValue(result.getCommandOutput(), CredsStoreEntity.class);
+            if(result==null || StringUtils.isBlank(result.getCommandOutput()) || result.getExitCode()!=1) {
```

Comment:
Exit code != 0

Reply:
Can you resolve Tushar's review comment ?
Return null when exitcode != 0